### PR TITLE
Fix delta dependency in IntelliJ

### PR DESCRIPTION
**Pull request by imback82**
### What is the context for this pull request?

I cannot compile/run tests inside IntelliJ.

### What changes were proposed in this pull request?

This PR proposes to fix the dependency for delta to compile/run tests in IntelliJ.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Manual testing.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -30,15 +31,16 @@ libraryDependencies ++= Seq(
...
 
   // Test dependencies
   "org.scalatest" %% "scalatest" % "3.0.5" % "test",
   "org.mockito" %% "mockito-scala" % "0.4.0" % "test",
   "org.apache.spark" %% "spark-catalyst" % sparkVersion.value % "test" classifier "tests",
   "org.apache.spark" %% "spark-core" % sparkVersion.value % "test" classifier "tests",
   "org.apache.spark" %% "spark-sql" % sparkVersion.value % "test" classifier "tests",
-  "org.apache.spark" %% "spark-hive" % sparkVersion.value % "test" classifier "tests"
+  "org.apache.spark" %% "spark-hive" % sparkVersion.value % "test" classifier "tests",
+  "io.delta" %% "delta-core" % deltaVersion % "test"
```
Comment (imback82) [inline build.sbt:43]: @sezruby @apoorvedave1 @pirz Can you compile/run tests inside InteillJ without this change (clear io.delta cache for ivy first)?

Comment (sezruby) [reply build.sbt:43]: @imback82 successfully compiled/ran tests without this change.

Comment (imback82) [reply build.sbt:43]: even after clearing the ivy cache?

Comment (imback82) [reply build.sbt:43]: image

Comment (sezruby) [reply build.sbt:43]: Yes I removed ~/.ivy2/cache and redownloaded all. If this change fixes the issue, then let's just merge it. It looks fine to me.

**Comment by imback82** [comment]
Looks like enabling auto-import build.sbt fixed the issue. Closing this.

**Pull request closed without merging**